fec.config.js

```diff
@@ -13,4 +16,22 @@ module.exports = {
    * Add additional webpack plugins
    */
   plugins: [],
-};
\ No newline at end of file
+  moduleFederation: {
+    exposes: {
+      './RootApp': path.resolve(__dirname, './src/AppEntry.tsx'),
+      './BookmarkedLearninResourcesWidget': path.resolve(
```

Comment:
@karelhala I think we have a typo (`Learnin` vs `Learning`)

Reply:
Good point!